client/src/modules/reports/generate/inventory_report/inventory_report.html

```diff
@@ -0,0 +1,101 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.STOCK.TITLE</h3>
+      <p class="text-info" translate>REPORT.STOCK.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+
+        <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate autocomplete="off">
+
+          <!-- choose date until  -->
+          <bh-date-editor
+            label="FORM.LABELS.UNTIL_PERIOD"
+            date-value="ReportConfigCtrl.dateTo">
+          </bh-date-editor>
+
+          <!-- delay -->
+          <div class="form-group col-xs-12 col-sm-6">
+            <label class="control-label" translate>FORM.LABELS.DELAY</label>
+              <div class="input-group">
+              <input class="form-control" type="number" step="1" name="delay" ng-model="ReportConfigCtrl.delay">
+              <span class="input-group-addon" translate>FORM.LABELS.MONTH</span>
+            </div>
+          </div>
+
+          <!-- purchase interval -->
+          <div class="form-group col-xs-12 col-sm-6">
+            <label class="control-label" translate>FORM.LABELS.PURCHASE_INTERVAL</label>
+            <div class="input-group">
+              <input class="form-control" type="number" step="1" name="purchase_interval" ng-model="ReportConfigCtrl.purchaseInterval">
+              <span class="input-group-addon" translate>FORM.LABELS.MONTH</span>
+            </div>
+          </div>
+
+          <!-- depot options -->
+          <div class="form-group">
```

Comment:
The correct way to specify a bootstrap checkbox  is demonstrated in the [documentation](http://getbootstrap.com/css/#checkboxes-and-radios).

```html
<div class="checkbox">
  <label>
    <input type="checkbox" value="">
    Option one is this and that&mdash;be sure to include why it's great
  </label>
</div>
```

This way causes the `form-group`'s label styling to by applied to the `<label>` and text doesn't always line up correctly.